src/include/nodes/primnodes.h

```diff
@@ -695,6 +695,10 @@ typedef struct FuncExpr
 	List	   *args;
 	/* token location, or -1 if unknown */
 	int			location;
+	/* bbf_pivot function required rewritted parsetrees */
+	List 	   *pivot_parsetree pg_node_attr(query_jumble_ignore, read_write_ignore, read_as(NULL));
```

Comment:
Please use a more general name like `extra_info` or `custom_list` so that other features can make use of this field as well. Same applies to other engine changes.

Reply:
Replaces `pivot_parsetree` and `pivot_extrainfo` with `Node * context` field.